### ISSUE TITLE
Internal error message box: Mini GUI => ShowProperties => Run tests of selected node

### ISSUE THREAD

**Issue by rowo360** [TestCentric/testcentric-gui#1093]
**Description:**
An internal error message box pops up in the described use case when working with the Mini-GUI and showing the property view.

**Reproduction:**
1. Switch to Mini-GUI view mode
2. Load project and select one test node
3. Use context menu to show test properties view
4. Click toolbar button 'Run all tests' or 'Run selected tests'
 => Error message box is shown

The same behavior can be observed with the 'View as XML" dialog.

**Additional information:**
Ultimately, this message box is displayed because there is a cross-thread operation resulting in an InvalidOperationException.

**Environment:**
- TestCentric Version 2.0.0-beta5
- OS: Windows 11
- Target Runtime for your test assembly: .NET Framework 4.8

**Comment by CharliePoole** [comment]
See if this exists in the latest dev build. I know I have fixed a lot of cross-thread calls since beta 5.

**Comment by CharliePoole** [comment]
@rowo360 Welcome to the project! Send me an email so we can talk more easily off-line. Mine is visible on my profile.

**Comment by rowo360** [comment]
Yes, I can confirm that this issue is also present with latest dev build.
